src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,472 @@
+		System.out.println("\n");
+	}
+
+
+	/**
+	 * This method is used to delete employee using user input
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void deleteUser(Scanner scanner) {
+
+		System.out.println("Do you want to delete User? Please enter the fullname::\n");
+		String fullname = ValidationUtils.emptyValidation(scanner, "Fullname");
+		employeeServiceImpl = new EmployeeServiceImpl();
+		boolean isDeleted = employeeServiceImpl.delete(fullname);
+		if (isDeleted) {
+			System.out.println(fullname + " is deleted successfully.\n");
+		}
+	}
+
+	/**
+	 * This method is used to terminate employee, delete use
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void terminateUser(Scanner scanner) {
+
+		System.out.println("Do you want to terminate User? Please enter the fullname::\n");
+		String fullname = ValidationUtils.emptyValidation(scanner, "Fullname");
+		employeeServiceImpl = new EmployeeServiceImpl();
+		boolean isTerminated = employeeServiceImpl.terminate(fullname);
+		if (isTerminated) {
+			System.out.println(fullname + " is terminated successfully.\n");
+		}
+
+	}
+
+	/**
+	 * This method is used to display edit employee options
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void editOptions(Scanner scanner) {
+
+		System.out.println("************************************************"
+			+ "\n" + "**   1. Full Name                             **" + "\n"
+			+ "**   2. Department                            **" + "\n"
+			+ "**   3. Address                               **" + "\n"
+			+ "************************************************" + "\n");
+	}
+
+	/**
+	 * This method is used to edit employee information
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void editEmployee(Scanner scanner, int id) {
+
+		System.out.println("Do you want to edit your information? Please choose the options::\n");
+		String fullname = null;
+		String department = null;
+		String address = null;
+		String message = null;
+		int choice = 0;
+		employeeServiceImpl = new EmployeeServiceImpl();
+		editOptions(scanner);
+		while (true) {
+			choice = ValidationUtils.numberValidation(scanner);
+			if (choice == 1) {
+				System.out.println("Please enter the full name::");
+				fullname =
+					ValidationUtils.emptyValidation(scanner, "Full Name");
+				break;
+			}
+			else if (choice == 2) {
+				System.out.println("Please enter the department::");
+				department =
+					ValidationUtils.emptyValidation(scanner, "Department");
+				break;
+			}
+			else if (choice == 3) {
+				System.out.println("Please enter the address::");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)